doc/Language/classtut.pod6

```diff
@@ -312,6 +227,35 @@ has Bool $.done is rw;
 The C<is rw> trait causes the generated accessor method to return a container
 so external code can modify the value of the attribute.
 
+=head2 C<is built> trait
+
+=for code
+has &!callback is built;
+
+X<|traits,is built>
+By default private attributes are not automatically set by the default constructor. (They are private after
+all.) In the above example we want to allow the user to provide the initial value but keep the attribute
```

Comment:
The thing is that it's not totally clear to me you've explained what private attributes are. There's a bunch of thing that's deleted, I would need to see how this one looks like, but my impression is that it's not covered.